index.html

```diff
@@ -34,5 +37,114 @@ <h1>OUR STRONG ORGANAISATION</h1>
+            <div class="box">
+               <h2>DO YOU KNOW WE CAN PROVIDE FOR YOU?</h2>
+               <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Suscipit eligendi possimus magnam molestias. Rem ex perspiciatis ullam sapiente eaque dolore delectus rerum laudantium doloremque ad expedita numquam deleniti laborum architecto ipsam omnis fuga, voluptatem nihil.</p>
+               <button class="button-secondary">Learn-More</button>
+            </div>
+         </div>
+         
+         <div class="col-sm-6 col-md-offset-1 grid-one">
+            <div class="row">
+               <div class="col-md-6 .grid-one-item">
+                  <img src="./img/Settings.png" alt="Setting">
+                  <span>Management</span>
+                  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet provident saepe perferendis a impedit sed quisquam corrupti molestiae, distinctio earum porro sint nulla quaerat?</p>
+               </div>
+              
+               <div class="col-md-6 .grid-one-item">
+                  <img src="./img/Pencil.png" alt="Pencil">
+                  <span>UI/UX Design</span>
+                  <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aliquid, at quam! Dolore, nobis eveniet modi illum natus minima consequatur sapiente unde veniam, nulla tempora!</p>
+               </div>
+            </div>
+
+            <div class="row">
+               <div class="col-md-6 .grid-one-item">
+                  <img src="./img/Diamond.png" alt="Diamond">
+                  <span>Logo/Branding</span>
+                  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt dolorem officia velit ad atque error voluptate magni. Quaerat, commodi repellendus in sunt porro enim.</p>
+               </div>
+               
+               <div class="col-md-6 .grid-one-item">
+                  <img src="./img/Truck.png" alt="Truck">
+                  <span>Animation</span>
+                  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Esse accusantium cumque ipsum? Error rerum cum natus? Asperiores ipsum esse maxime laudantium omnis, minima molestias.</p>
+               </div>
+            </div>
+         </div>
+      </div>
+
+      <div class=" col-sm-12 col-md-8 feature-products">
+         <div class="feature-products-text">
+         <h3>Feature-Products</h3>
+         <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt fuga minima aperiam voluptates, esse a laborum debitis natus maiores?</p>
+         </div>
+      
+         <div class="feature-products-nav">
+            <a class="feature-products-nav-item" href="#">ALL</a>
+            <a class="feature-products-nav-item" href="#">PRINT TEMPLATE</a>
+            <a class="feature-products-nav-item" href="#">WEB TEMPLATE</a>
+            <a class="feature-products-nav-item" href="#">USER INTERFACE</a>
+            <a class="feature-products-nav-item" href="#">MOCK-UP</a>
```

Comment:
you didn't close your `div.feature-products-nav` tag, please check it before push, it shouldn't work this way